### PR TITLE
add comment feature cucumber ready

### PR DESCRIPTION
**Pull request by irchik**
Need review

### CODE REVIEW

File: features/functionality/articles/comments/adding.feature [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@wip
+@ok
```
Comment (romikoops) [inline features/functionality/articles/comments/adding.feature]: remove this tag

File: features/step_definitions/article_steps.rb [hunk 2]
```diff
@@ -21,6 +21,17 @@
   step "I submit form on new article page"
   step 'user logged out'
 end
+Given /^comment to (.+) article with parameter:$/ do |article,table|
+
```
Comment (romikoops) [inline features/step_definitions/article_steps.rb]: What for this blank definition?

File: features/step_definitions/article_steps.rb [hunk 3]
```diff
@@ -21,6 +21,17 @@
...
+end
+
+Given /^opened (.+) article page$/ do |article|
+  ArticleListPage.open
+  ArticleListPage.given.click_article_button(article)
```
Comment (romikoops) [inline features/step_definitions/article_steps.rb]: What does Article Button mean?

File: spec/bvt/functionality/articles/comments/add_comment_spec.rb [hunk 4]
```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+feature "Adding Comment" do
+  background "Create article, user, comment" do
+    @article = Gen.article
+    @comment = Gen.comment
+    @user1 = Gen.user
```
Comment (romikoops) [inline spec/bvt/functionality/articles/comments/add_comment_spec.rb]: Why user1?

File: spec/bvt/functionality/articles/comments/add_comment_spec.rb [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+    log_in_as_admin
+    create_article(@article)
+    sign_up_as(@user1)
+    log_in_as(@user1)
+    article_open(@article)
```
Comment (romikoops) [inline spec/bvt/functionality/articles/comments/add_comment_spec.rb]: verb+noun - general rule of method naming

Comment (irchik) [inline spec/bvt/functionality/articles/comments/add_comment_spec.rb]: fixed

File: spec/bvt/functionality/articles/comments/add_comment_spec.rb (hunk 4 above)
Comment (irchik) [inline spec/bvt/functionality/articles/comments/add_comment_spec.rb]: fixed

File: features/step_definitions/article_steps.rb (hunk 3 above)
Comment (irchik) [inline features/step_definitions/article_steps.rb]: it was before me. I`ve rename already

File: features/step_definitions/article_steps.rb (hunk 2 above)
Comment (irchik) [inline features/step_definitions/article_steps.rb]: for next feature. fixed.

File: features/functionality/articles/comments/adding.feature (hunk 1 above)
Comment (irchik) [inline features/functionality/articles/comments/adding.feature]: fixed

File: config/default.yml [hunk 6]
```diff
@@ -22,7 +22,7 @@
  maximized_window: true
 
  # Specify one of the following drivers: selenium, selenium_dev, webkit, poltergeist, sauce, testingbot, phantomjs
- driver: phantomjs
+ driver: poltergeist
```
Comment (romikoops) [inline config/default.yml]: please do not modify default.yml. Revert this change
